crates/bitwarden-fido/Cargo.toml

```diff
@@ -30,7 +30,7 @@ coset = { version = "0.3.7" }
 itertools = "0.13.0"
 log = ">=0.4.18, <0.5"
 p256 = { version = ">=0.13.2, <0.14" }
-passkey = { git = "https://github.com/bitwarden/passkey-rs", rev = "c48c2ddfd6b884b2d754432576c66cb2b1985a3a" }
+passkey = { git = "https://github.com/bitwarden/passkey-rs", rev = "ae08e2cb7dd3d44d915caed395c0cdc56b50fa27" }
```

Comment:
we also have passkey defined in `bitwarden-core`, so we'd need to update that one as well to not bring two copies of the library.